src/benchmark/inference_benchmark.py

```diff
@@ -36,13 +42,13 @@ def cli_argument_parser():
     return args
 
 
-def inference_benchmark(executor_type, test_list, output_handler, log):
+def inference_benchmark(executor_type, test_list, output_handler, log, cpp_benchmark_path=None):
     process_executor = Executor.get_executor(executor_type, log)
     for test in test_list:
-        test_process = ProcessHandler.get_process(test, process_executor, log)
+        test_process = ProcessHandler.get_process(test, process_executor, log, cpp_benchmark_path)
         test_process.execute()
 
-        log.info('Saving test result in file\n')
```

Comment:
Why do you remove it? it's for making log more readable

Reply:
Didn't remove, it just moved below